helm/config.go

```diff
@@ -179,6 +217,10 @@ func (scalar *Scalar) String() string {
 
 // Value returns the value of a scalar node.
```

Comment:
The comment should mention that this method returns string values *without* the surrounding quotes.

Reply:
https://github.com/SUSE/fissile/pull/282/commits/0ed98d295f2ada6bfa5590e0688a379f9f61d580